### PR TITLE
SIG-2096 Document revised return URL

### PR DESCRIPTION
**Pull request by Nuss9**
Deletes the old documentation on the return URL and updates the document with the newer version where the return URL contains a description of the user action.

### CODE REVIEW

**Comment by Nuss9** [comment]
@MrJoe ready

File: index.md [hunk 1]
```diff
@@ -105,8 +105,21 @@ For more information about postbacks and calculating the checksum and the body o
...
-?sh_id=transactionid&status=status_of_transaction&reference=your_reference
+When the user signs or rejects the transaction we will redirect the browser to the return URL.
+The return URL will have one of the following parameters, determined by the user action:
+
+*   **User signs :**
+    [yourReturnUrl]signerstatus=signed
+*   **User rejects :**
+    [yourReturnUrl]signerstatus=rejected
+*   **Other :**
+    [yourReturnUrl]signerstatus=
```
Comment (MrJoe) [inline index.md]: please add or [yourReturnUrl  (ie no signer status parameter)

Comment (MrJoe) [reply index.md]: ps. isn't it sh_signerstatus=... ?

**Comment by Nuss9** [comment]
screen shot 2018-08-01 at 16 36 53

File: index.md [hunk 2]
```diff
@@ -105,8 +105,21 @@ For more information about postbacks and calculating the checksum and the body o
...
-?sh_id=transactionid&status=status_of_transaction&reference=your_reference
+When the user signs or rejects the transaction we will redirect the browser to the return URL.
+The return URL will have one of the following parameters, determined by the user action:
+
+*   **User signs :**
+    https://www.example.org/[yourReturnUrl]?sh_signerstatus=signed
+*   **User rejects :**
+    https://www.example.org/[yourReturnUrl]?sh_signerstatus=rejected
+*   **Other :**
+    https://www.example.org/[yourReturnUrl]?sh_signerstatus=
```
Comment (MrJoe) [inline index.md]: https://www.example.org/[yourReturnUrl] or https://www.example.org/[yourReturnUrl]?sh_signerstatus=

File: index.md [hunk 3]
```diff
@@ -105,8 +105,21 @@ For more information about postbacks and calculating the checksum and the body o
...
+
+The user actions provide an indication on what the user chose to do.
+Depending on the choice, you might want to show a different page.
+Once the user signed or rejected the transaction, the server will process the input.
+Therefore, the return URL will always be available before the transaction status has been updated.
+Once the server has completed processing, a new postback will be send with the new transaction status.
```
Comment (MrJoe) [inline index.md]: You _must_ validate the actual status using the API and you _should not_ trust the sh_signerstatus for any important processing.

**Pull request merged**